data-otservbr-global/lib/core/storages.lua

```diff
@@ -3079,6 +3079,13 @@ GlobalStorage = {
 		OutburstHealth = 60181,
 		OutburstChargingKilled = 60182,
 	},
+	WorldBoard = {
+		NightmareIsle = {
+			ankrahmunNorth = 60191,
+			darashiaNorth = 60192,
+			darashiaWest = 60193,
```

Comment:
favor alterar para o padrão PascalCase e ajustar seus usos.. obrigado.
```suggestion
			AnkrahmunNorth = 60191,
			DarashiaNorth = 60192,
			DarashiaWest = 60193,
```